iceoryx_utils/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -114,6 +112,11 @@ UnixDomainSocket& UnixDomainSocket::operator=(UnixDomainSocket&& other) noexcept
 {
     if (this != &other)
```

Comment:
We all implemented the move assignment in the wrong fashion. Please read: https://isocpp.org/wiki/faq/assignment-operators

and we need to add here the move assignment for the creation pattern. Additionally, we have to implement it as well (which is not yet being done but I work on it in a different PR). Could you please add here only the explicit call to the move assignment of the creation:
```cpp
DesignPattern::Creation<UnixDomainSocket, UnixDomainSocketError>::operator=(std::move(other));
```

Reply:
Imporant point, thanks for raising it! Fixed.